### ISSUE TITLE
Add a (per-platform?) "default rat" field to users

### ISSUE THREAD

**Issue by dewiniaid** [FuelRats/api.fuelrats.com#105]
If a dispatcher uses `!assign Client Ratnickname`, Mecha needs to be able to determine which of a user's rats to assign to a case.

One tool that can help with this would be if the user has a 'default rat' attribute (which I thought we had, but I'm not seeing in source) -- Mecha would assign the correct rat to the case by platform.

This should probably default to the "first" rat belonging to the user for each platform, and should never be null unless they have no rats on that platform.

**Comment by xlexi** [comment]
Is this still necessary? @Marenthyu

**Comment by Marenthyu** [comment]
Currently, it is using the stripped IRC name as CMDRname (as is, it removes everything behind '[') and then uses the /rats?CMDRname= endpoint. This is suboptimal, but i can change it. Thing with the platform is a different one as i want to keep the method generalized, but i can add a platform parameter which defaults to "None" and uses the first rat found of a user, otherwise only checks the platform-specific rats available. I should be able to use the platform parameter though in some cases when a case is also given, but we need to get it into dispatch's heads to use the !pc and !xb commands to actually set the cases platforms.

TL;DR:
No, it's not needed, but i am not using the users endpoint rn anyway.
